### DISCUSSION TITLE
Publish Helm Charts

### DISCUSSION THREAD

**Discussion by arpa42** [zwave-js/zwave-js-ui#2713] [Ideas]
Hi, 
I see the Kubernetes deployements files. It would be nice to have a Helm Chart published on a repository like artifacthub.io

**Comment by robertsLando** [comment]
cc @chrisns ?

**Reply by chrisns** [reply]
I've a passionate dislike of helm, but acknowledge some folk like it.
Is this a feature request and/or are you volunteering to do it @ArnaudPalgen ?

**Reply by arpa42** [reply]
Sorry for the late reply. It was more of a feature request, as I am far from being a Helm Chart expert. However, I can try :)

**Reply by robertsLando** [reply]
I never used them, this is the first time I see them 😆

**Reply by chrisns** [reply]
I'm jealous of your ignorance 😂

**Reply by varet80** [reply]
Good day! I am going to create one. anybody want to be tester? will be soon under artifactshub.io

**Comment by varet80** [comment]
@ArnaudPalgen initial commit: https://github.com/k8sonlab/publiccharts/tree/add_zwave_js/charts/zwave-js-ui
did not test it yet! and is not going to be released!
Once I test it I will

The first version will have no mqtt installation (support for) and will be a simple chart

Later will add more, I am going on a short vacation (till tuesday) will probably publish it wednesday in artifacthub.io
